README.md

```diff
@@ -116,10 +116,14 @@ make container
    ```
    $ helm install harbor-scanner-aqua ./helm/harbor-scanner-aqua \
                   --namespace harbor \
+                  --set aqua.version=4.5 \
+                  --set aqua.registry.server=aquasec.azurecr.io \
+                  --set aqua.registry.username=$AQUA_REGISTRY_USERNAME \
+                  --set aqua.registry.password=$AQUA_REGISTRY_PASSWORD \
                   --set image.tag=dev \
                   --set scanner.logLevel=trace \
-                  --set scanner.aqua.user=$AQUA_USER \
-                  --set scanner.aqua.password=$AQUA_PASSWORD \
+                  --set scanner.aqua.user=$AQUA_CONSOLE_USERNAME \
+                  --set scanner.aqua.password=$AQUA_CONSOLE_PASSWORD \
```

Comment:
In the config table below there's documentation of `SCANNER_AQUA_USER` which I suppose is what's set by `scanner.aqua.user` (ditto password) but maybe we need to also document what the Registry username & password are

Reply:
We have docs for environment variables used by the adapter service in the config table below and indeed `SCANNER_AQUA_USER` env corresponds to the `scanner.aqua.user`.

There're no corresponding `AQUA_REGISTRY_USERNAME` for adapter service. It only makes sense as a Helm value and is used to create K8S secret for pulling scanner image in the init container.

Hence, the aqua.registry.username Helm value is only documented in helm/harbor-scanner-aqua/README#parameters